api/src/services/expenditureCategories/expenditureCategories.ts

```diff
@@ -19,6 +21,39 @@ export const expenditureCategory: QueryResolvers['expenditureCategory'] = ({
   })
 }
 
+export const createOrSkipInitialExpenditureCategories = async () => {
+  const expendtitureCategoriesData = [
+    {
```

Comment:
Might be nice to have these in a fixtures file or similar just for cleanliness? Don't feel strongly though 🤷‍♀️

Reply:
Or, minus that, perhaps a doc comment on the function about what this is used for / it should be called from the seed scripts only

Reply:
I moved it to the `constants.ts` file so the function is cleaner now 👍